### PR TITLE
Add Elixir support

### PR DESCRIPTION
**Pull request by zenbro**
Compiler template for Elixir.

### CODE REVIEW

File: autoload/SingleCompile/templates/elixir.vim [hunk 1]
```diff
@@ -0,0 +1,27 @@
+" Copyright (C) 2010-2012 Hong Xu
```
Comment (xuhdev) [inline autoload/SingleCompile/templates/elixir.vim]: @zenbro Thanks for the PR. Maybe you wanna update this line?

**Comment by xuhdev** [comment]
Sorry that I didn't see the PR until now---I have my comments as a line note.

**Comment by zenbro** [comment]
@xuhdev done! :smiley:

**Comment by xuhdev** [comment]
@zenbro I actually meant whether you wanna to use your name as well... But I'm fine with my name on it. The year should be soly 2015 (not 2012-2015).

**Comment by zenbro** [comment]
@xuhdev sorry, now it's 2015.

**Comment by xuhdev** [comment]
OK, thanks!

**Pull request merged**